api/envoy/extensions/resource_monitors/downstream_connections/v3/downstream_connections.proto

```diff
@@ -12,7 +12,7 @@ option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/res
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Downstream connections]
-// [#extension: envoy.resource_monitors.downstream_connections]
+// [#extension: envoy.resource_monitors.global_downstream_max_connections]
```

Comment:
Just to make sure I understand this correctly, prior to this PR, if a user used `envoy.resource_monitors.global_downstream_max_connections` everything worked (but it was inconsistent with the docs).
If that's the case, then LGTM. The other way around might be more problematic, as it may break current users.

Reply:
yes, this is the case

Reply:
Thanks!